sensu-plugins-vault.gemspec

```
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+
+require 'date'
+require_relative 'lib/sensu-plugins-vault'
+
+Gem::Specification.new do |s| # rubocop:disable Metrics/BlockLength
+  s.authors                = ['Sensu-Plugins and contributors']
+  s.date                   = Date.today.to_s
+  s.description            = 'Sensu Vault plugins'
+  s.email                  = '<sensu-users@googlegroups.com>'
+  s.executables            = Dir.glob('bin/**/*.rb').map { |file| File.basename(file) }
+  s.files                  = Dir.glob('{bin,lib}/**/*') + %w[LICENSE README.md CHANGELOG.md]
+  s.homepage               = 'https://github.com/sensu-plugins/sensu-plugins-vault'
+  s.license                = 'MIT'
+  s.metadata               = { 'maintainer'         => 'sensu-plugin',
+                               'development_status' => 'active',
+                               'production_status'  => 'unstable - testing recommended',
+                               'release_draft'      => 'false',
+                               'release_prerelease' => 'false' }
+  s.name                   = 'sensu-plugins-vault'
+  s.platform               = Gem::Platform::RUBY
+  s.post_install_message   = 'You can use the embedded Ruby by setting EMBEDDED_RUBY=true in /etc/default/sensu'
+  s.require_paths          = ['lib']
+  s.required_ruby_version  = '>= 2.3.0'
+  s.summary                = 'Sensu plugins for vault'
+  s.test_files             = s.files.grep(%r{^(test|spec|features)/})
+  s.version                = SensuPluginsVault::Version::VER_STRING
+
+  s.add_runtime_dependency 'sensu-plugin', '~> 2.7'
```

I realize this was probably contributed before 3.0 was released but I think we should update it:
```suggestion
  s.add_runtime_dependency 'sensu-plugin', '~> 3.0'
```

---

I just released 4.0 so i will update for that.